content/docs/user-guide/gems/reference/debug/imgui.md

```diff
@@ -1,8 +1,102 @@
+
+By default, discrete input mode is off. It can be turned on in the ImGui O3DE menu, or with the [CVAR](#cvars) `imgui_DiscreteInputMode`
+
+When this mode is enabled, ImGui will be given a 2nd visibility state, at this point, when toggling ImGui visibility via the HOME button or L3/R3 on a game controller, it will toggle through three states instead of just on and off.
+
+1. ImGui is off - ImGui is not visible,
+2. All input goes to ImGui - ImGui is visible and receiving all input.
+3. All input goes to the Game - ImGui is visible but input is going to the game. This allows ImGui profiling tools and others to be visible on screen while interacting with O3DE.
+
+> You can look at the upper right hand corner of the ImGui Main Menu bar to see the current state of ImGui input. You can also interact with this menu for some input tips.
+
+![ImGui Input Status](/images/user-guide/gems/reference/debug/imgui_input_and_status.png)
+
+## Using ImGui in your component
+
+You will need to:
+
+1. Ensure the ImGui Gem is active in your project. See the O3DE guide on [Adding Gems in a project](https://www.o3de.org/docs/user-guide/project-config/add-remove-gems/).
+2. Sign up for ImGui::ImGuiUpdateListenerBus::Handler or use the `ImGuiLYCommonMenu` class as a base.
+3. Inside the handler perform your ImGui actions
+
+```
+        void MyImGuiComponent::OnImGuiUpdate()
+        {
+            // .. do imgui stuff ...
+        }
+```
+
+## Example O3DE Imgui tools
+
+O3DE ships with many ImGui tools so the options you see will be dependent on the specific gems active in your project.
+
+* [Debugging PhysX](https://o3de.org/docs/user-guide/interactivity/physics/debugging/)
+* [Atom Rendering Profiler](https://www.o3de.org/docs/atom-guide/atom-sample-viewer/debug-menu-samples/)
+* [Multiplayer helpers](https://www.o3de.org/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running/#running-locally-using-imgui-options)
+
+## CVARs
+
+ImGui uses the following Console variables (CVARs) either at runtime via the console, or by placing them in configuration. See the general [CVAR guide](https://www.o3de.org/docs/user-guide/appendix/cvars/) for more information on configuring CVARs.
+
+| Name                             | Description                                                                                                                                                               | 
+|----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| imgui_AutoEnableComponents       | _Deprecated_                                                                                                                                                              |
```

Comment:
Would it be better to leave deprecated cvars out of the table?

Reply:
I added them as they are discoverable via the console/console cvar window but seem to have no current impact. So we could omit them if preferred, leave the description blank for future maintainers, or mark them in someway to suggest they may not be functional.

Happy to take suggestion here.